helm/dagster/templates/ingress.yaml

```diff
@@ -58,6 +58,7 @@ spec:
                   number: {{ $_.Values.dagsterWebserver.service.port | default 80 }}
           {{- range $_.Values.ingress.dagsterWebserver.succeedingPaths }}
           - path: {{ .path }}
+            pathType: {{ $_.Values.ingress.dagsterWebserver.pathType }}
```

Comment:
I think this should be '.pathType' not '$_.Values.ingress.dagsterWebserver.pathType'? Since it can vary per succeeding path. If you look at the example in the values.yaml it looks like this:

```
    # Different http paths to add to the ingress after the default webserver path
    # Example:
    #
    # succeedingPaths:
    #   - path: "/*"
    #     pathType: ImplementationSpecific
    #     serviceName: "ssl-redirect"
    #     servicePort: "use-annotation"
```

Reply:
(it also matches the corresponding line for the 'read only' webserver on line 88)